Sources/MockoloFramework/Models/ThrowingKind.swift

```diff
@@ -0,0 +1,32 @@
+enum ThrowingKind: Equatable {
+    case none
+    case any
+    case `rethrows`
+    case typed(errorType: String)
+
+    var hasError: Bool {
+        switch self {
+        case .none:
+            return false
+        case .any:
+            return true
+        case .rethrows:
+            return true
+        case .typed(let errorType):
+            return errorType != .neverType && errorType != "Swift.\(String.neverType)"
```

Comment:
`throws(Never)` means no throws. Without this consideration, there would be warnings in the generated code.